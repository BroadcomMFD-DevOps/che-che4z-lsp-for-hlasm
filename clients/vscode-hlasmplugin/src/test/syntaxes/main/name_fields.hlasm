ORDSYM         INSTR OP
ORD&VARSYM     INSTR OP

=LBL.LBL       INSTR OP
!@#$%^*()"     INSTR OP    
123456789      INSTR OP

''             INSTR  OP REM
'LBL           INSTR  OP REM
LBL'           INSTR  OP REM
'LBL'          INSTR  OP REM
'LBL'          INSTR  OP REM'
'LB L'         INSTR  OP REM
'LB L'         INSTR  OP REM'
'L''B'         INSTR  OP REM
'L'' B'        INSTR  OP REM

LA' BE'L       INSTR OP REM
LA' BE''       INSTR OP REM
LA' BEL''L     INSTR OP REM
