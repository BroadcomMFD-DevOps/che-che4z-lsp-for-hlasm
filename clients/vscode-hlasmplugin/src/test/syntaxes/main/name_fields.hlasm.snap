>ORDSYM INSTR OP
#^^^^^^ source.hlasm support.class.hlasm
#      ^ source.hlasm
#       ^^^^^ source.hlasm entity.name.function.hlasm
#            ^ source.hlasm
#             ^^ source.hlasm punctuation.definition.tag.hlasm
>ORD&VARSYM INSTR OP
#^^^ source.hlasm support.class.hlasm
#   ^^^^^^^ source.hlasm support.class.hlasm variable.other.readwrite.hlasm
#          ^ source.hlasm
#           ^^^^^ source.hlasm entity.name.function.hlasm
#                ^ source.hlasm
#                 ^^ source.hlasm punctuation.definition.tag.hlasm
>
>=LBL.LBL INSTR OP
#^^^^^^^^ source.hlasm support.class.hlasm
#        ^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^^ source.hlasm punctuation.definition.tag.hlasm
>
>''       INSTR  OP REM
#^^ source.hlasm support.class.hlasm
#  ^^^^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm comment.hlasm
>'LBL     INSTR  OP REM
#^^^^ source.hlasm support.class.hlasm
#    ^^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm comment.hlasm
>LBL'     INSTR  OP REM
#^^^^ source.hlasm support.class.hlasm
#    ^^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm comment.hlasm
>'LBL'    INSTR  OP REM
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm comment.hlasm
>'LBL'    INSTR  OP REM'
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^^ source.hlasm comment.hlasm
>'LB L'   INSTR  OP REM
#^^^^^^ source.hlasm support.class.hlasm
#      ^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm comment.hlasm
>'LB L'   INSTR  OP REM'
#^^^^^^ source.hlasm support.class.hlasm
#      ^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^^ source.hlasm comment.hlasm
>'L''B'   INSTR  OP REM
#^^^^^^ source.hlasm support.class.hlasm
#      ^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm comment.hlasm
>'L'' B'  INSTR  OP REM
#^^^^^^^ source.hlasm support.class.hlasm
#       ^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^ source.hlasm
#                ^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm comment.hlasm
>
>LA' BE'L       INSTR OP REM
#^^^^^^^^ source.hlasm support.class.hlasm
#        ^^^^^^^ source.hlasm
#               ^^^^^ source.hlasm entity.name.function.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm punctuation.definition.tag.hlasm
#                       ^ source.hlasm
#                        ^^^ source.hlasm comment.hlasm
>LA' BE''       INSTR OP REM
#^^^^^^^^ source.hlasm support.class.hlasm
#        ^^^^^^^ source.hlasm
#               ^^^^^ source.hlasm entity.name.function.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm punctuation.definition.tag.hlasm
#                       ^ source.hlasm
#                        ^^^ source.hlasm comment.hlasm
>LA' BEL''L     INSTR OP REM
#^^^^^^^^^^ source.hlasm support.class.hlasm
#          ^^^^^ source.hlasm
#               ^^^^^ source.hlasm entity.name.function.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm punctuation.definition.tag.hlasm
#                       ^ source.hlasm
#                        ^^^ source.hlasm comment.hlasm
>
>
>!@#$%^*()"     INSTR OP    
#^^^^^^^^^^ source.hlasm support.class.hlasm
#          ^^^^^ source.hlasm
#               ^^^^^ source.hlasm entity.name.function.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm punctuation.definition.tag.hlasm
#                       ^^^^ source.hlasm
>123456789      INSTR OP
#^^^^^^^^^ source.hlasm support.class.hlasm
#         ^^^^^^ source.hlasm
#               ^^^^^ source.hlasm entity.name.function.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm punctuation.definition.tag.hlasm
>utf8🧿          INSTR OP
#^^^^^^ source.hlasm support.class.hlasm
#      ^^^^^^^^^^ source.hlasm
#                ^^^^^ source.hlasm entity.name.function.hlasm
#                     ^ source.hlasm
#                      ^^ source.hlasm punctuation.definition.tag.hlasm
>utf8🧿_TEXT     INSTR OP
#^^^^^^^^^^^ source.hlasm support.class.hlasm
#           ^^^^^ source.hlasm
#                ^^^^^ source.hlasm entity.name.function.hlasm
#                     ^ source.hlasm
#                      ^^ source.hlasm punctuation.definition.tag.hlasm
>$utf8㎛_🧽__߽    INSTR OP
#^^^^^^^^^^^^ source.hlasm support.class.hlasm
#            ^^^^ source.hlasm
#                ^^^^^ source.hlasm entity.name.function.hlasm
#                     ^ source.hlasm
#                      ^^ source.hlasm punctuation.definition.tag.hlasm
>&utf8㎛_🧽__߽    INSTR OP
#^^^^^ source.hlasm support.class.hlasm variable.other.readwrite.hlasm
#     ^^^^^^^ source.hlasm support.class.hlasm
#            ^^^^ source.hlasm
#                ^^^^^ source.hlasm entity.name.function.hlasm
#                     ^ source.hlasm
#                      ^^ source.hlasm punctuation.definition.tag.hlasm
>utf8㎛_🧽__߽&var INSTR OP
#^^^^^^^^^^^ source.hlasm support.class.hlasm
#           ^^^^ source.hlasm support.class.hlasm variable.other.readwrite.hlasm
#               ^ source.hlasm
#                ^^^^^ source.hlasm entity.name.function.hlasm
#                     ^ source.hlasm
#                      ^^ source.hlasm punctuation.definition.tag.hlasm
>